docs/core/integration-quality-scale/rules/iqs019.md

```diff
@@ -108,6 +108,11 @@ class MyConfigFlow(ConfigFlow, domain=DOMAIN):
         )
 ```
 
+:::info
+If you are using DHCP discovery, and you want to receive discovery flows for updated IP addresses, be sure to register the MAC address in the device info and set `registered_devices` to `true` in the manifest.
```

Comment:
I think this should go in another rule. I think I said that already.

Reply:
That is actually a good point 👍